### ISSUE TITLE
Cannot download vide from self paced course

### ISSUE THREAD

**Issue by redstoneleo** [ivan-dyachenko/coursera-downloader#24]
e.g.
https://www.coursera.org/learn/human-computer-interaction/lecture/pE6EB/human-computer-interaction

**Comment by EMCP** [comment]
same issue for me, different course

**Comment by redstoneleo** [comment]
@EMCP I wrote an extension ,it works well, you can download it here http://pan.baidu.com/s/1hq1v88G

**Comment by AhmedKamal** [comment]
@redstoneleo  The link doesn't work , can you please provide us with a new working link ?

**Comment by redstoneleo** [comment]
I wrote an extension ,it works well, you can download it here  http://mathjoy.tumblr.com/post/130547895523/mediadownloader

**Comment by AhmedKamal** [comment]
I dont think it would be useful for me as long as it requires playing the video before downloading :(
